docs/deployment.md

```diff
@@ -82,7 +82,31 @@ site:
 If you are using Git, add the `_build` folder to your `.gitignore` so that it is not tracked. This folder contains auto-generated assets that can easily be re-built -- for example in a Continuous Integration system like GitHub Actions.
 :::
 
-### Deploying to a Static Web Server
+(deploy:base-url)=
+### Custom domains and the base URL
+
+When deploying static HTML to the web, you may have to specify a Base URL so that links resolve properly.
+This happens when you serve your MyST site from a *subfolder* of a parent URL.
+For example:
+
+- `mysite.org/`: **Does not** require a `BASE_URL` to be set, because the root site is where your MyST site lives.
```

Comment:
This is super helpful, chris. i may be confused about the text above as it seems to contradict what is written VERY CLEARLY here.